server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/AbstractRecipientRewriteTableMailet.java

```diff
@@ -19,22 +19,28 @@
 

```

Comment:
Subclass and inheritance is a bad way of sharing code. The Abstract class have no substance, and no exact definition of its scope here.

On the other hand, if you extract common functionnalityinto an other service, you : 

 - makes it possible to use functionnality without extending
 - you makes it easier to test common service than abstract class
 - It's easier to read as what is being used is explicitly named.